experimental/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ All notable changes to experimental packages in this project will be documented
 
 * fix(otlp-transformer): remove type dependency on Long #3022 @legendecas
 * fix(grpc-exporter): use non-normalized URL to determine channel security #3019 @pichlermarc
+* fix(fetch): pass spanResponse to _endSpan() [#3037](https://github.com/open-telemetry/opentelemetry-js/pull/3037) @Ugzuzg
```

Comment:
This describes the technical internals but not the actual bug being fixed that an end-user could care about. The CHANGELOG file is meant to be a resource for end users.